index.md

```diff
@@ -52,8 +52,29 @@ because your past self doesn't answer email.
+
+**Create analysis-friendly data.** Well-prepared raw data can still present many barriers to analysis and visualization. For example, columns in the dataset may not represent the variables that are most conducive to analysis. There are (at least!) two classes of problems:
+
+  1. Columns that contain more than one variable's worth of information. For example, the inclusion of units is problematic, e.g., "3.4 kg". The presence of "kg" will cause most analytical environments to read this in as character data, whereas you'll probably want to do numeric things with it, like take averages and use in plots. This should be split into two variables, the mass "3.4" and the units "kg" (or the units should be recorded in the variable name and/or metadata). When in doubt, try to make each variable correspond to an atomic, imminently usable piece of information.
+  2. Multiple columns that -- only when taken together -- contain one variable's worth of information. This is characteristic of data that has been laid out for human eyeballs or for manual data entry. For example, there might be one row per field site and then columns for measurements made at each of several time points. For data entry and inspection, it is convenient to store this in a "short and wide" form, but for most analyses it will be advantageous to gather these columns into a variable of measurements, accompanied by a companion variable indicating the time point.
+
+The goal of all this variable splitting, combining, spreading, and gathering is to create so-called "tidy data", which can be a powerful accelerator for analysis ([Tidy Data][wickham-tidy], [Nine simple ways][white-simple-reuse]). Other considerations: this is a good place to make sure your data will play nicely with your analytical environment and plans. For example, reformat a date-time to match one of those recognized automatically or standardize the casing on a character variable. This pass through the data likely does not change the amount of data, but may dramatically alter its form.
+
+*This may still need something about observations and rows and the notion of a key?*
+
+**Choose your friends wisely.** Marshal complementary data in files with the same high standard for openness and simplicity as above. Frequently the raw data does not and indeed cannot fully explain itself. For example, if each well of a microtitre plate is used to study a specific gene knockout, the prepared raw data will have a `well` variable taking on values like "A1" and "G12", but the plate reader cannot possibly know what's in each well. That information will be absent from the primary data file. You must create a supplementary file in order to look this up. Head off future join headaches by taking care to use the same names and codes when variables in two datasets refer to the same thing and will be used for merging or table lookup.
```

Comment:
complementary data and supplementary file, could this simply be replaced with the term 'metadata'?